src/Autoloading/BootstrapFilesIncluder.php

```diff
@@ -87,10 +88,16 @@ private function resolveAbsoluteBootstrapFilePaths(array $extensionConfigFiles):
         return $absoluteBootstrapFilePaths;
     }
 
-    private function tryRequireFile(string $bootstrapFile): void
+    /**
+     * PHPStan container mimics:
+     * https://github.com/phpstan/phpstan-src/blob/34881e682e36e30917dcfa8dc69c70e857143436/src/Command/CommandHelper.php#L513-L515
+     */
+    private function tryRequireFile(string $bootstrapFile, ?Container $phpstanContainer = null): void
```

Comment:
I think variable should be `$container`, same with what phpstan does, as it used via `use()` in closure to be used inside require_once, not as parameter in closure.

Reply:
Could you add the diff snippet of what you mean?

Reply:
Yes:

```suggestion
    private function tryRequireFile(string $bootstrapFile, ?Container $container = null): void
```